keyboards/keychron/c2_pro/ansi/rgb/rules.mk

```diff
@@ -0,0 +1,11 @@
+# Build Options
+#   change yes to no to disable.
+#
+
+# Enter lower-power sleep mode when on the ChibiOS idle thread
+OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
+
+# custom matrix setup
+CUSTOM_MATRIX = lite
+
+SRC += matrix.c
```

Comment:
```suggestion
# This file intentionally left blank
```
Since `ansi/white/rules.mk` has the exact same code, best to consolidate this into somewhere like `c2_pro/rules.mk`, with the exception of 
```mk
# custom matrix setup
CUSTOM_MATRIX = lite
```
as that can be moved into `c2_pro/info.json` inside the `"matrix_pins" parameter`.